### DISCUSSION TITLE
Strange soil response to penetration of a rigid body with prescribed velocity

### DISCUSSION THREAD

**Discussion by albuquerqueraniel** [Anura3D/Anura3D_OpenSource#96] [Q&A] [answered]
Dear all,

I'm trying to simulate the driving of a tube in clay. Here is some information about my model:

- Axissymetric model;
- The tube is modeled as a rigid body. 
- Saturated soil;
- There are 4 (for this is the maximum number of materials that one can define contact with a rigid body) soil layers ;
- The soils diameter is equal to 10 D, where D is the tube's outer diameter;
- A prescribed velocity of about (D/2)m/s is defined at the MP of the rigid body;

There is the system at 0.0 s:
![geometria](https://user-images.githubusercontent.com/45863863/231790295-4e4c3393-8c0d-4dfc-ae26-ecd48ff7fbb3.png)

After 0.1 s this is the MP velocity:
![velocity](https://user-images.githubusercontent.com/45863863/231791985-897040f6-59f8-4510-bc53-de3669103189.png)

![velocity2](https://user-images.githubusercontent.com/45863863/231792348-004039bb-5c92-433f-b2b1-66f7b8ab5dd2.png)

The maximum velocity at the soil is 52x the prescribed velocity of the rigid body. It doesn't make sense to me. It doesn't seem to have a physical meaning. 

Could you help me to point out possible reasons for this soil behavior and how to solve it?

Best regards,
Raniel

**Comment by FrancescaCeccato** [comment]
Are you using contact and moving mesh? These are necessary in this problem.

**Reply by albuquerqueraniel** [reply]
Yes, I'm using both. The moving mesh reference material is the rigid body. Here are the points shared by the moving mesh and the compressing mesh:

![MOVING MESH](https://user-images.githubusercontent.com/45863863/235036028-ddd752e7-35ae-4be8-ba07-45885e2a5776.png)

And here is the contact associated to the rigid body, with the friction properties for the 4 soil layers:

![contato](https://user-images.githubusercontent.com/45863863/235036518-c85a12c9-3c17-4fe8-8526-195999903fb7.png)

**Some more information on what I observed:**

I tried to run the model with absorbing boundaries at the bottom boundary (for I just can define 1 reference material for the absorbing boundaries and couldn't define for the lateral boundaries), but a particle was leaving the mesh through this region:

![particle leaving the mesh](https://user-images.githubusercontent.com/45863863/235038027-76b3e2aa-d58e-4f11-bfd5-a5b5b2d98b72.png)

I tried several values for alpha and delta, but couldn't stop the particle from leaving. The particle leave the mesh at the load step 2. The particle take longer to leave the mesh (time step number increases) if I reduce the courant number, but even when I set courant number = 0.1 the particle leaved the domain.

When I used rigid boundaries, the disturbance on the soil starts on the region where the particle leaves the mesh in the simulation with absorbing boundaries: 

![deforma1](https://user-images.githubusercontent.com/45863863/235054895-a640e92e-aefd-44ec-a5fb-a64938d80898.png)
![deforma2](https://user-images.githubusercontent.com/45863863/235054917-2d801e8e-b735-4b0d-938c-857b1bcced9a.png)

A hole is formed in this region and the disturbance propagates through the soil.

Here are the parameters I am using in the CPS:

$$COURANT_NUMBER
0.98

$$COMPUTATION_METHOD
MPM-MIXED

$$GRAVITY_ACCELERATION
9.81
$$GRAVITY_VECTOR
0.0 -1.0 0.0
$$GRAVITY_LOAD
step  1.0  1.0
$$SOLID_TRACTION
off  0.0  1.0
$$SOLID_TRACTION_B
off  0.0  1.0
$$LIQUID_PRESSURE
off  0.0  1.0
$$LIQUID_PRESSURE_B
off  0.0  1.0
$$PRESCRIBED_VELOCITY
step  1.0  1.0
$$HYDRAULIC_HEAD
off 0.0
$$APPLY_SEEPAGE_FACE
0
$$APPLY_INFILTRATION
0
$$QUASISTATIC_CONVERGENCE
0
$$TOLERATED_ERROR_SOLID
0.01 0.01
$$TOLERATED_ERROR_LIQUID
0.01 0.01
$$MAXIMUM_TIME_STEPS
10000
$$MASS_SCALING
0 1.0
$$HOMOGENEOUS_LOCAL_DAMPING
0 0.75
$$BULK_VISCOSITY_DAMPING
0 0.06 1.2
$$FIX_SOLID_SKELETON
0
$$STRAIN_SMOOTHING
1
$$APPLY_SMOOTHENING_LIQUID_PRESSURE_INCREMENT
0
$$CONTACT_FORMULATION
1
$$INITIAL_WATER_PRESSURE
0.0
$$APPLY_POROSITY_UPDATE
0
$$INITIAL_VELOCITY
0
$$RESET_DISPLACEMENTS
0
$$REMOVE_FIXITIES
0 0 0
$$K0_PROCEDURE
1
$$SURFACE_ELEVATION
0.0
$$INITIAL_VERTICAL_LOAD_K0
0.0
$$EXCAVATION
0
$$SUBMERGED_CALCULATION
1 1
$$APPLY_OBJECTIVE_STRESS
0
$$DEGREE_OF_FILLING
0.9
$$NUMBER_OF_ACTIVE_ELEMENTS
0

**Reply by FrancescaCeccato** [reply, marked as answer]
Thank you for tring these solutions. How fast is moving your body? If it goes form 0 velocity to a certain value, there may be dynamic waves disturbing the solution. One option could be start the motion with a linear increase of velocity. Also bulk viscosity damping or a bit of local damping may help.
